### DISCUSSION TITLE
How to uninstall ?

### DISCUSSION THREAD

**Discussion by ZeSimon** [microsoft/GDK#38] [Q&A]
Hello, I need to uninstall every GDK and PGDK package from my machine. There is no uninstall executable, is there any command line I could use ?
Thanks

**Comment by TriceHelix** [comment]
- Open `C:/Windows/System32/appwiz.cpl`.
- Find the GDK (and related packages) within the list.
- Click on the entry and press "Uninstall", which will launch the uninstaller. The uninstaller has a GUI which you have to navigate, and admin permissions are required to remove the GDK.
